src/SprykerEco/Zed/Payone/Communication/Plugin/Oms/Command/CaptureCommandPlugin.php

```diff
@@ -35,8 +35,12 @@ public function run(array $orderItems, SpySalesOrder $orderEntity, ReadOnlyArray
         $paymentTransfer = new PayonePaymentTransfer();
         $paymentTransfer->setFkSalesOrder($orderEntity->getSpyPaymentPayones()->getFirst()->getFkSalesOrder());
         $captureTransfer->setPayment($paymentTransfer);
-        $captureTransfer->setAmount($this->getOrderTransfer($orderEntity)->getTotals()->getGrandTotal());
-        $captureTransfer->setOrder($this->getOrderTransfer($orderEntity));
+
+        $orderTransfer = $this->getOrderTransfer($orderEntity);
+        $orderTransfer = $this->getFacade()->distributePrices($orderTransfer);
```

Comment:
Please move the price distribution under the existing facade method.
Check similar places.

Reply:
Fixed